### PR TITLE
 add prime number code

### PR DESCRIPTION
**Pull request by nathan90**

### CODE REVIEW

File: exercises/005_prime.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//Enter a number and find if it's prime
+const readlineSync = require("readline-sync");
+const num = readlineSync.question("Enter a Positive Integer:");
+
+const isPrime = (num) => {
```
Comment (kuism) [inline exercises/005_prime.js:5]: this is wrong

Comment (kuism) [reply exercises/005_prime.js:5]: Logic is wrong.

Comment (kuism) [reply exercises/005_prime.js:5]: Try value 25.

Comment (nathan90) [reply exercises/005_prime.js:5]: Fixed the issue

**Pull request merged**